src/eduid/userdb/fixtures/users.py

```diff
@@ -43,219 +42,177 @@
-    orcid=dashboard_orcid,
-    ladok=dashboard_ladok,
-)
 
-mocked_user_standard_2 = User(
-    meta=Meta(version=ObjectId("987654321098765432103210")),
-    eppn="babba-labba",
-    user_id=ObjectId("901234567890123456789012"),
-    given_name="John",
-    surname="Smith",
-    display_name="John Smith",
-    identities=IdentityList(),
-    language="en",
-    entitlements=["urn:mace:eduid.se:role:admin", "urn:mace:eduid.se:role:student"],
-    phone_numbers=PhoneNumberList(),
-    mail_addresses=MailAddressList(elements=[johnsmith_example_org, johnsmith2_example_org]),
-    credentials=CredentialList(elements=[signup_password]),
-)
+class UserFixtures:
+    @property
+    def mocked_user_standard(self) -> User:
+        return User(
+            meta=Meta(version=ObjectId("987654321098765432103210")),
+            eppn="hubba-bubba",
+            user_id=ObjectId("012345678901234567890123"),
+            given_name="John",
+            surname="Smith",
+            display_name="John Smith",
+            identities=IdentityList(elements=[verified_nin_identity]),
+            language="en",
+            entitlements=["urn:mace:eduid.se:role:admin", "urn:mace:eduid.se:role:student"],
+            phone_numbers=PhoneNumberList(
+                elements=[dashboard_primary_phone, dashboard_verified_phone, dashboard_unverified_phone]
+            ),
+            mail_addresses=MailAddressList(
+                elements=[johnsmith_example_com, johnsmith2_example_com_old, johnsmith3_example_com_unverified]
+            ),
+            credentials=CredentialList(elements=[signup_password]),
+            orcid=dashboard_orcid,
+            ladok=dashboard_ladok,
+        )
 
-new_completed_signup_user_example = User(
-    meta=Meta(version=ObjectId("987654321098765432103210")),
-    eppn="hubba-fooo",
-    user_id=ObjectId("000000000000000000000002"),
-    given_name="John",
-    display_name="John Smith",
-    surname="Smith",
-    subject=SubjectType.PERSON,
-    language="en",
-    modified_ts=datetime.fromisoformat("2017-01-04T16:47:30"),
-    tou=ToUList(elements=[signup_2016_v1]),
-    terminated=None,
-    mail_addresses=MailAddressList(elements=[johnsmith3_example_com]),
-    identities=IdentityList(),
-    phone_numbers=PhoneNumberList(elements=[dashboard_primary_phone, dashboard_unverified_phone]),
-    credentials=CredentialList(elements=[signup_password_2]),
-    entitlements=[],
-    locked_identity=LockedIdentityList(),
-)
+    @property
+    def new_user_example(self) -> User:
+        return User(
+            meta=Meta(
+                cleaned={
+                    CleanerType.SKV: datetime.fromisoformat("2017-01-04T16:47:30"),
+                },
+            ),
+            eppn="hubba-bubba",
+            user_id=ObjectId("012345678901234567890123"),
+            given_name="John",
+            display_name="John Smith",
+            surname="Smith",
+            subject=SubjectType("physical person"),
```

Comment:
Varför inte använda enum syntaxen (SujectType.PERSON)?

Reply:
Jag flyttade bara koden in i en klass, så jag vet inte.

Möjligen har någon någon gång haft tanken att det är ett implicit test av att strängen "physical person" (som vi säkert har i databasen) faktiskt kan laddas korrekt, men jag skulle föredra ett mycket mer explicit test av det snarare än att göra det i förbifarten i våra fixtures...